src/hatti/views/table.cljs

```diff
@@ -13,9 +13,9 @@
             [hatti.views.record]
             [hatti.shared :as shared]
             [hatti.utils :refer [click-fn generate-html hyphen->camel-case
-                                 last-url-param safe-regex url]]
+                                 last-url-param safe-regex url any?]]
             [cljsjs.slickgrid-with-deps]))
-
+(enable-console-print!)
```

Comment:
Might be a good idea to add an automatic check for this. I've done it a few times before myself.